app/retail/templates/jtbd/connect.html

```diff
@@ -1,7 +1,388 @@
-projects: <pre>{{ projects | pprint }}</pre><br>
+{% comment %}
+Copyright (C) 2019 Gitcoin Core
 
-alumni: <pre>{{ alumni | pprint }}</pre><br>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
 
-hackathons: <pre>{{ hackathons | pprint }}</pre><br>
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
 
-testimonial: <pre>{{ testimonial | pprint }}</pre>
\ No newline at end of file
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not,see
+<http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n static to_date %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {% include 'shared/head.html' %}
+  {% include 'shared/cards_pic.html' %}
+  <meta name="title" content="Connect | Gitcoin">
+  <meta name="description" content="Connect">
+
+
+  <style>
+
+    section {
+        margin-top: 7em;
+    }
+
+    .img-accordion .collapsing {
+      transition: none !important;
+    }
+
+    .accordion:not(.collapsed) {
+      pointer-events: none;
+    }
+
+    @media screen and (max-width: 768px) {
+      .img-accordion > div {
+        left: 0;
+      }
+    }
+
+    @media screen and (min-width: 768px) {
+      .row > .col-gaps-2:nth-of-type(1) {
+        padding-right: 0.5rem!important;
+      }
+      .row > .col-gaps-2:nth-of-type(2) {
+        padding-left: 0.5rem!important;
+      }
+
+      .row > .col-gaps-3:nth-of-type(1) {
+        padding-right: 2rem!important;
+      }
+      .row > .col-gaps-3:nth-of-type(2) {
+        padding-left: 1rem!important;
+        padding-right: 1rem!important;
+      }
+      .row > .col-gaps-3:nth-of-type(3) {
+        padding-left: 2rem!important;
+      }
+
+      .row > .col-gap-border {
+        box-sizing: border-box;
+        border: 0.5rem solid transparent;
+        background-clip: padding-box;
+      }
+
+      .btn-explore {
+        position: absolute;
+        bottom: 4em;
+        min-width: 15rem;
+      }
+    }
+
+  </style>
```

Comment:
^ Q: given that we have compressor and stuff -> why do we wanna use css within HTML ? why not throw it into it's own css and have it compressed ?

Reply:
Yeah, it's because we've all been working on these at the same time, once we've merged everything to landings we can collate all the css into a new file 👍